tutorials/celestia-da.md

```diff
@@ -34,16 +34,22 @@ ignite scaffold chain gm --address-prefix gm --no-module
 Install the Rollkit app to ignite:
 
 ```bash
+cd $HOME/gm
 ignite app install github.com/ignite/apps/rollkit@rollkit/v0.2.0
 ```
 
-Next, move to the `gm` directory and add the Rollkit app:
+Add the Rollkit app:
 
 ```bash
-cd $HOME/gm
 ignite rollkit add
 ```
 
+Build the Rollkit chain:
```

Comment:
is this necessary @yarikbratashchuk ?

Reply:
No, rollkit start does it, let's remove it

Reply:
@a26nine